src/columns/fast-formula-parser.ts

```diff
@@ -7,15 +7,26 @@ const run: glide.Column = (formula, ...params) => {
 
     const parser = new FormulaParser({
         onCell: ({_, row, col}) => {
-            return data[row - 1][col - 1];
+            
+            try {
+                const res = JSON.parse(data[row - 1][col - 1]);
+                return typeof res === 'number' ? res : undefined;
```

Comment:
Don't assume the value is a string.  Only parse if it's a string.

Also, if you can't parse, or it's not a string, or it doesn't parse to a number, return the original value.  Formulas can operate on other values, too.